mods/cnc/weapons/explosions.yaml

```diff
@@ -1,7 +1,28 @@
+^DamagingExplosion:
+	Warhead@1Dam: SpreadDamage
+		Spread: 426
+		Damage: 40
+		Versus:
+			None: 90
+			Wood: 75
+			Light: 60
+			Heavy: 25
+		DamageTypes: Prone50Percent, TriggerProne, ExplosionDeath
+	Warhead@2Eff: CreateEffect
+		Explosions: poof
+		ImpactSounds: xplos.aud
+		TargetSearchRadius: 0
+	Warhead@3Smu: LeaveSmudge
+		SmudgeType: Crater
+		InvalidTargets: Vehicle, Structure, Wall, Husk, Trees, Creep
+
 FlametankExplode:
+	Inherits: ^DamagingExplosion
 	Warhead@1Dam: SpreadDamage
 		Spread: 1c0
 		Damage: 115
+		Versus:
```

Comment:
In this particular case that was intentional, I don't see why a flame-based explosion should be this effective vs. heavy targets and assumed this was an oversight when this weapon was added.